Include/object.h

```diff
@@ -124,12 +124,45 @@ typedef struct {
 #define Py_TYPE(ob)             (_PyObject_CAST(ob)->ob_type)
 #define Py_SIZE(ob)             (_PyVarObject_CAST(ob)->ob_size)
 
+
+/* [RFC] Should we enable Immortal Instances by Default? */
+#define Py_IMMORTAL_INSTANCES
```

Comment:
Possible alternative: Py_IMMORTAL_OBJECTS.  I think it is more accurate.  Long ago, only some Python objects were instances.  Within the cpython codebase, "object" is more common than "instance".

Reply:
Makes sense, totally in favor of keeping the standard across the codebase!